Components/Comments/CommentAdder.jsx

```diff
@@ -30,12 +30,7 @@ function AddComment(props) {
     setNewComment("");
   };
 
-  // const postComment = () => {
-  //   articlesApi.post(`/articles/${article_id}/comments`, {
-  //     username: "jessjelly",
-  //     body: newComment,
-  //   });
-  // };
+  
 
```

Comment:
The only changes made by this PR seems to be commented code that is being removed 🤔